### PR TITLE
requirements.txt

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
This PR adds pip requirements to the unix deployment doc proposed in https://github.com/openmicroscopy/openmicroscopy/pull/4301/files adds 

cc: @sbesson @manics

### CODE REVIEW

File: omero/developers/Web/Deployment.txt [hunk 1]
```diff
@@ -66,7 +66,7 @@ Using WSGI (Unix/Linux)
 -----------------------
 
 For convenience you may wish to run a web server under your local user account
-instead of using a system server for testing. Install Nginx
+instead of using a system server for testing. Install Nginx and Gunicron
```
Comment (mtbc) [inline omero/developers/Web/Deployment.txt]: Gunicorn

**Comment by joshmoore** [comment]
Changes make sense, however the page is a bit tough to read: [Prerequisites](https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/unix/install-web.html#prerequisites) mentions Django so someone might try to install it there, but the details on *how* to install django and the WSGI server are under [Deployment](https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/unix/install-web.html#deployment-unix-linux). The [Intro](https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/unix/install-web.html#omero-web-deployment) also has some of this. Is a simplification possible?

NB: there's also no mention of Django **1.6** on https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/install-web/install-iis.html (other than a "note" at the very bottom)

**Comment by hflynn** [comment]
Don't worry about the Windows install web page, I'm opening a new PR and will cover that there.

**Comment by aleksandra-tarkowska** [comment]
ok thx, sorry about my comment on iis PR I misread

**Comment by aleksandra-tarkowska** [comment]
@joshmoore:
> NB: there's also no mention of Django 1.6 on https://www.openmicroscopy.org/site/support/omero5.2-staging/sysadmins/windows/install-web/install-iis.html (other than a "note" at the very bottom) -- in case it's of use: openmicroscopy/openmicroscopy#4319

there is https://github.com/openmicroscopy/ome-documentation/blob/develop/omero/sysadmins/windows/install-web.txt#L37

**Comment by aleksandra-tarkowska** [comment]
In general it is true that Prerequisites don't look right because it is tricky to add all ``pip install {requirements-...} `` to the global section. Maybe prerequisites should be moved to particular deployment page

File: omero/sysadmins/unix/install-web/install-apache.txt [hunk 2]
```diff
@@ -6,14 +6,25 @@ OMERO.web Apache and mod_wsgi deployment (Unix/Linux)
...
+    or
+    :djangodoc:`how to upgrade Django to 1.8 <topics/install/#remove-any-old-versions-of-django>`
+
+OMERO can automatically generate a
 configuration file for your web server. The location of the file
 will depend on your system, please refer to your web server's manual.
 See :ref:`customizing_your_omero_web_installation_unix`
 for additional customization options.
 
-Install `mod\_wsgi <http://www.modwsgi.org/>`_.
```
Comment (aleksandra-tarkowska) [inline omero/sysadmins/unix/install-web/install-apache.txt]: Please do not remove this. This is on pip installed!

**Comment by aleksandra-tarkowska** [comment]
just discussed with @hflynn looks like it reads much better if we push Django prerequisite to each individual install page.

**Comment by hflynn** [comment]
I've ended up touching all the same pages so am closing this in favour of #1327

**Comment by hflynn** [comment]
See https://github.com/hflynn/ome-documentation/commit/699f3e4b6fddb84147a86bc3021c79ac2ba58a20

**Pull request closed without merging**